index.js

```diff
@@ -309,52 +514,109 @@ Ycb.prototype = {
         var usedDimensions = {};
         var usedValues = {};
         var contexts = {};
+        var intervals = {};
         configLoop:
         for(var i=0; i<config.length; i++) {
-            if (config[i].settings) {
-                var setting = config[i].settings;
-                if(setting.length === 0 ) {
+            var configObj = config[i];
+            if(!configObj || configObj.constructor !== Object) {
+                logBundleWarning(i, 'non-object config', JSON.stringify(configObj));
+                continue;
+            }
+            if(!configObj.settings) {
+                if(!configObj.dimensions) {
```

Comment:
dimensions is not required right?, the old syntax should still work

Reply:
This check prevents an erroneous warning log from the dimensions object in the bundle.